pkg/config/config.go

```diff
@@ -92,6 +95,14 @@ var (
 	////////////////////////////////////
 )
 
+func getBoolConfig(configKey string, defaultBool bool) bool {
+	defaultValue := "false"
```

Comment:
we have defaultValue and defaultBool?

Reply:
defaultValue is string and defaultBool is bool. 
This is to avoid human error in the function input when setting the default value for the Boolean variable.